### DISCUSSION TITLE
Configure UI not showing up.

### DISCUSSION THREAD

**Discussion by deltamelter** [BottlecapDave/HomeAssistant-OctopusEnergy#433] [General]
Looked at how to add a mini last week and the dialog was there...  now it arrived and I come to add it to the integration and the UI shows this:
![image](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/27061628/9914e4b1-6348-47b7-97d4-12fb36143c78)
How do I add the mini? Which option here do I need to check/populate?

**Comment by deltamelter** [comment]
![image](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/27061628/6bd5be74-306d-4307-b18c-0d29a00c956a)

**Comment by BottlecapDave** [comment]
Do you have any errors in your logs or have you updated your integration recently? It looks like HA has had issues loading the translations for some reason. A reload of the integration or restart of HA should fix your issue.

p.s. Home mini is the first checkbox on the first screenshot.

**Reply by deltamelter** [reply]
Updated to the latest integration in readiness for the mini arriving and also too the opportunity to upgrade the HA version too

Looking at general logs in HA I see this, doesit say anything to you?:
`Logger: homeassistant.helpers.integration_platform
Source: loader.py:855
First occurred: 6:59:09 PM (6 occurrences)
Last logged: 7:52:50 PM

Unexpected error importing octopus_energy/system_health.py
Unexpected error importing octopus_energy/logbook.py
Unexpected error importing octopus_energy/media_source.py
Unexpected error importing octopus_energy/energy.py
Unexpected error importing octopus_energy/hardware.py
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/integration_platform.py", line 50, in _async_process_single_integration_platform_component
    platform = integration.get_platform(platform_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/loader.py", line 838, in get_platform
    cache[full_name] = self._import_platform(platform_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/loader.py", line 855, in _import_platform
    return importlib.import_module(f"{self.pkg_path}.{platform_name}")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'custom_components.octopus_energyv81.recorder'`

**Reply by BottlecapDave** [reply]
Hello. Those errors look like something has gone wrong with the download, as it's trying to access things my integration doesn't use. I'm not sure why it's looking for 'custom_components.octopus_energyv81.recorder'`. Did you install the integration manually or via HACS? Have you tried removing and re-installing the integration?

**Comment by NOLBwithaD** [comment]
Hi there BottlecapDave,
Apologies for troubling you and for possibly being thick. :-(.  I’m installing the Octopus Energy integration into HA and have this dialog box open.  The first field (above “The hours you require in decimal format”) is expecting an entry, which states Required, but there is no hint as to what should go in that field.  Can you please advise, and perhaps add a hint into the script for the dialog box for other users.  I’m guessing it might be my API, but don’t want to insert that into an arbitrary field.  Thanks for your excellent project.

**Reply by BottlecapDave** [reply]
Nope not being silly. I assume you are referring to adding a target rate sensor? If so, it looks like a translation has been missed for that field. The first field should be the name you want to give the sensor (e.g. washing machine)

**Comment by NOLBwithaD** [comment]
Hi Dave,
That makes perfect sense.

That seemed to work.  Thank you for your speedy response.

Neil
